public/style.css

```diff
@@ -15,3 +28,57 @@
 ul {
   list-style: none;
 }
+
+#list-of-rules{
+  list-style:square;
+  text-align: left;
+  margin-left: 20px;
+  margin-right: 40px;
+  background-color: #DBE2EF;
+}
+
+#list-of-rules li{
+  background-color: #DBE2EF;
+  line-height: 1.5em;
+}
+
+#start-quiz-button {
+  background-color: #112D4E;
+  border-color: #112D4E;
+  color: #F9F7F7;
+  width: auto;
+  height: auto;
+  border-radius: 10px;
+  font-size: 200%;
+  padding: 10px 15px;
+}
+
+#start-quiz-button:hover {
+  background-color: #3F72AF;
+  border-color: #3F72AF;
+  box-shadow: 0 12px 16px 0 #112D4E, 0 17px 50px 0 #DBE2EF;
+  cursor: pointer;
+}
+
+#rules-of-the-quiz{
+  border: solid 2px #3F72AF;
+  border-radius: 2px;
+  background-color: #DBE2EF;
+  margin: 30px auto;
+  font-size: 35px;
+}
+
+#rules-of-the-quiz h3{
+  background-color: #DBE2EF;
+}
+
+#welcome-h1 {
+  font-size: 500%;
```

Comment:
That would give desired result but the best practice is to use pixels instead.

Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/font-size#pixels